client/src/components/JobParameters/JobParameters.vue

```diff
@@ -56,16 +56,19 @@ export default {
     props: {
         jobId: {
             type: String,
+            default: null,
         },
         datasetId: {
             type: String,
+            default: null,
         },
         datasetType: {
             type: String,
             default: "hda",
         },
         param: {
             type: String,
+            default: undefined,
```

Comment:
why some are null and others are undefined?

Reply:
I prefer all of them to be always `null` as default (or an actual value), however this component relies on `param` to be `undefined` and I did not want to change it yet. See below: https://github.com/galaxyproject/galaxy/blob/241ca75bb62588c6814dc44bc01d484cabb6911d/client/src/components/JobParameters/JobParameters.vue#L96